pykafka/balancedconsumer.py

```diff
@@ -209,7 +213,8 @@ def checker():
 
     @property
     def partitions(self):
-        return self._consumer.partitions if self._consumer else None
+        return self._consumer.partitions if (
+            self._consumer and self._consumer._running) else None
```

Comment:
I'm not sure I like checking `self._consumer._running` here. The consumer can still be holding partitions even if it's not running (though they may not match zookeeper). I'm pretty sure this check is what's making the tests fail after my master merge, too.